README.md

```diff
@@ -14,169 +12,202 @@
 7. [Slack/Gitter/Stackoverflow discussion](#ask-questions-start-a-discussion)
 
 ### Introduction
-[SnappyData](https://github.com/SnappyDataInc/snappydata) aims to deliver real time operational analytics at interactive speeds with commodity infrastructure and far less complexity than today. SnappyData fulfills this promise by
-- Enabling streaming, transactions and interactive analytics in a single unifying system rather than stitching different solutions—and
-- Delivering true interactive speeds via a state-of-the-art approximate query engine that leverages a multitude of synopses as well as the full dataset. SnappyData implements this by deeply integrating an in-memory database into Apache Spark.
+[SnappyData](https://github.com/SnappyDataInc/snappydata) aims to deliver real time operational analytics at interactive
+speeds with commodity infrastructure and far less complexity than today.
```

Comment:
"far less complexity than today" - looks a little vague. Can we be more specific as to what complexities are involved and how are we getting rid of those?